server/routes/api/delete/deleteUserRoutes.ts

```diff
@@ -0,0 +1,75 @@
+import { Router, Request, Response } from "express";
+import { validationResult, param } from "express-validator";
+
+import Handler from "./deleteUserHandler";
+
+const deleteRouter = Router();
+
+const handler = new Handler();
+const BABYSITTER_INVALID_INPUT_ERROR = "babysitterId must be provided and a number";
+const PARENT_INVALID_INPUT_ERROR = "parentId must be provided and a number";
+const ACTIVE_END_TIMESTAMP = '9999999999';
```

Comment:
you can import this from the global folder and then you don't need to have ACTIVE_END_TIMESTAMP here

Reply:
done
global.ts file created by Eli